src/components/home/Event.jsx

```diff
@@ -18,7 +18,7 @@ const Event = ({ start, end, location, name, descripton, color }) => {
       <div className="p-3 w-5/6 text-ewb-black ">
         <div className={`text-2xl ${COLORS[color].text}`}>{name}</div>
         <div className="text-base">{location}</div>
-        <div className="text-base font-normal">{descripton}</div>
```

Comment:
keep it base in a bigger screen. so you can do something like "text-sm md:text-base" so the text will be small on small screen and be base on screen lager than middle